src/OpenTelemetry.Collector.Dependencies/OpenTelemetry.Collector.Dependencies.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>net46;netstandard2.0</TargetFrameworks>
```

Comment:
I have never tested it for net46 http libraries. Should test project be also supporting net46 then?

Reply:
IIRC the guidance is that if the package is expected to be used by .NET Framework and the package needs `netstandard2.0`, it should always multi-target with at least, `net461` too. Due to the issues in dependency resolution caused by `ns2.0` and netfx.

But if the goal here is to support a lower netfx that also does the job.

Reply:
You are right. on .net 46 6 tests failing here with 
"The I/O operation has been aborted because of either a thread exit or an application request"

![image](https://user-images.githubusercontent.com/578954/72180618-956bd700-33f8-11ea-8a89-566886e0c9cb.png)

reverted for the moment, because don't ready to debug now.
But the same feedback applicable for some other tests like exporters.

Reply:
> IIRC the guidance is that if the package is expected to be used by .NET Framework and the package needs `netstandard2.0`, it should always multi-target with at least, `net461` too. Due to the issues in dependency resolution caused by `ns2.0` and netfx.
> 
> But if the goal here is to support a lower netfx that also does the job.

Looks like we should not only consolidate versions, but also bump to net461 (first .net version that can use .net standard 2.0)

Reply:
I wouldn't say we need to consolidate versions.

We should target the lowest possible while taking into consideration the maintenance burden.
So far it's been OK to keep the API in `net45`, the SDK `net46` and Exporters etc would depend on their dependencies, that's why, as pointed out here, Redis here targets `net461`.

Reply:
well, at least we have conversation log why it was done in this way.
dotnet 4.6 in api reset back to 4.5
everything else- are exporters. We can reject PR fully or at least merge tests with net46 added, because previously some exporters were .net standard only, while other been dualframework